dev/snippets/config/templates/stateful_widget.tmpl

```diff
@@ -0,0 +1,32 @@
+{{description}}
```

Comment:
Where is it documented to snippet authors that if they use the "stateful_widget" template, how to structure their snippet so as to use the "description", "code-preamble", and "code" blocks?